### PR TITLE
Heron's Formula #1155 added in all 4 languages

### PR DESCRIPTION
**Pull request by swastik-akhil**
## 👨‍💻 Changes proposed

- [x] Added Heron's formula code in C, C++, Java, Python.

## 📝Note

1.  My code follows the code style of this project.
2.  This PR does not contain plagiarized content.
3.  The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

I have myself reviewed all 4 files and all of them are working correctly.

## 📷 Screenshots
herons jpg

### CODE REVIEW

**Comment by swastik-akhil** [comment]
@Kumar-laxmi I have completed this issue and hence I would like to draw your attention towards it.

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Place java program in ```Java/Maths``` folder

File: heronsFormula.java
Comment (zugzwang03) [inline heronsFormula.java:1]: Put this under Java directory.

**Review by zugzwang03 — CHANGES_REQUESTED** [review]

**Comment by swastik-akhil** [comment]
I have done the changes, kindly check @Kumar-laxmi @zugzwang03

**Review by Arun9739 — APPROVED** [review]

File: C++/Maths/heronsFormula.CPP
Comment (zugzwang03) [inline C++/Maths/heronsFormula.CPP:1]: make this .cpp

**Review by 072arushi — APPROVED** [review]

**Review by zugzwang03 — APPROVED** [review]

**Comment by swastik-akhil** [comment]
> Place java program in `Java/Maths` folder

Resolved

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**